baseapp/abci.go

```diff
@@ -537,9 +537,24 @@ func gRPCErrorToSDKError(err error) error {
 	}
 }
 
+func errOnNegativeHeight(height int64) error {
```

Comment:
if you prefix a function with `err` it sounds like we want to create an error. Here we want to assert / check. How about renaming to `assertNegativeHeight`? (or `checkNegativeHeight`)

Reply:
The context of the name is err on negative height, but sure, checkNegativeHeight can work. Thanks.